markdown/preface.markdown

```diff
@@ -190,9 +194,8 @@ print >>sys.stderr, "fatal error"  # Python 2
 print("fatal error", file=sys.stderr)  # Python 3
 ```
 
-Ah, that looks a bit more worthwhile.
-What the hell is going on in that Python 2 statement?
-The authors don't rightly know.
+That change certainly seems worthwhile; and what is going on in the
+Python 2 version anyway?  The authors don't rightly know.
```

Comment:
@stefanv we haven't resolved the whether adding more to worthwhile is worthwhile. =P I argue that it is.

As before, I'm also inclined to leave the Doug Adams quote, but can be persuaded otherwise. @hdashnow, do you want to make a stand on this issue or let it slide? (And I also don't like the addition of "anyway".)